sdk/python/tests/test_offline_online_store_consistency.py

```diff
@@ -193,34 +193,51 @@ def check_offline_and_online_features(
     driver_id: int,
     event_timestamp: datetime,
     expected_value: Optional[float],
+    full_feature_names: bool,
 ) -> None:
     # Check online store
     response_dict = fs.get_online_features(
-        [f"{fv.name}:value"], [{"driver": driver_id}]
+        [f"{fv.name}:value"],
+        [{"driver": driver_id}],
+        full_feature_names=full_feature_names,
     ).to_dict()
 
-    if expected_value:
-        assert abs(response_dict[f"{fv.name}__value"][0] - expected_value) < 1e-6
+    if full_feature_names:
+        if expected_value:
+            assert abs(response_dict[f"{fv.name}__value"][0] - expected_value) < 1e-6
+        else:
+            assert response_dict[f"{fv.name}__value"][0] is None
     else:
-        assert response_dict[f"{fv.name}__value"][0] is None
+        if expected_value:
+            assert abs(response_dict["value"][0] - expected_value) < 1e-6
+        else:
+            assert response_dict["value"][0] is None
 
     # Check offline store
     df = fs.get_historical_features(
         entity_df=pd.DataFrame.from_dict(
             {"driver_id": [driver_id], "event_timestamp": [event_timestamp]}
         ),
         feature_refs=[f"{fv.name}:value"],
+        full_feature_names=full_feature_names,
     ).to_df()
 
-    if expected_value:
-        assert abs(df.to_dict()[f"{fv.name}__value"][0] - expected_value) < 1e-6
+    if full_feature_names:
+        if expected_value:
+            assert abs(df.to_dict()[f"{fv.name}__value"][0] - expected_value) < 1e-6
+        else:
+            df = df.where(pd.notnull(df), None)
+            assert df.to_dict()[f"{fv.name}__value"][0] is None
     else:
-        df = df.where(pd.notnull(df), None)
-        assert df.to_dict()[f"{fv.name}__value"][0] is None
+        if expected_value:
+            assert abs(df.to_dict()["value"][0] - expected_value) < 1e-6
+        else:
+            df = df.where(pd.notnull(df), None)
+            assert df.to_dict()["value"][0] is None
 
 
 def run_offline_online_store_consistency_test(
-    fs: FeatureStore, fv: FeatureView
+    fs: FeatureStore, fv: FeatureView, ffn: bool
```

Comment:
can we spell out the argument?

Reply:
Will do! I'm assuming you would like me to fully spell out the original `fs` and `fv` arguments also?

Reply:
both `fs` and `fv` are indicative of the primitives we use. Most people will understand what they are. `ffn` was mysterious to me.